utils/remote/common.py

```diff
@@ -83,7 +85,11 @@ def __init__(self, hostname, port=22, timeout=5, cmd_timeout=20,
         self.hostkey_change_cb = hostkey_change_cb
 
     def _connect_transport(self, transport, username=None, password=None):
-
+        """
+        :param transport: paramiko.Transport instance
+        :param username: SSH username
+        :param password: Either a plain-text password or a file object containing a private key
+        """
```

Comment:
We should continue to use the google docstring format.

Reply:
Gotcha...